### ISSUE TITLE
Get deposit address API

### ISSUE THREAD

**Issue by ssldhl** [shideneyu/kraken_client#7]
Is there any way I can implement https://www.kraken.com/help/api#deposit-addresses to get deposit address from kraken?

**Comment by shideneyu** [comment]
Yes sure. You can add a few lines here: https://github.com/shideneyu/kraken_client/blob/master/lib/kraken_client/endpoints/private.rb#L24

And it will be implemented. If you plan to do a pull request, please do not forget to update the gem version, readme, and specs please, thanks :)

Otherwise I'll do that when I got the time.
